### PR TITLE
Add Ghost Browser.app v1.0.3.3

### PR DESCRIPTION
**Pull request by straxhaber**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/ghost-browser.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'ghost-browser' do
+  version '1.0.3.3'
+  sha256 '3ffe5736563c25e834c23ba9f62e0188bb575e55df6370a56f3f13a90de40bad'
+
+  # ghostbrowser.s3.amazonaws.com was verified as official when first introduced to the cask
+  url 'https://ghostbrowser.s3.amazonaws.com/GhostBrowser-1.0.3.3.dmg'
```
Comment (miccal) [inline Casks/ghost-browser.rb]: Please change this to `url "https://ghostbrowser.s3.amazonaws.com/GhostBrowser-#{version}.dmg"`.

Comment (straxhaber) [inline Casks/ghost-browser.rb]: Okay. Should I submit a new pull request? This is my first time editing a commit after a pull request has been generated.

Comment (miccal) [inline Casks/ghost-browser.rb]: You can just edit the original commit if you like.

**Comment by straxhaber** [comment]
@miccal I edited the commit and rebased. Did I do it correctly? Let me know if there is anything else left to do.

File: Casks/ghost-browser.rb (hunk 1 above)
Comment (straxhaber) [inline Casks/ghost-browser.rb]: @miccal I edited the commit and rebased. Did I do it correctly? Let me know if there is anything else left to do.

Comment (miccal) [inline Casks/ghost-browser.rb]: You need to use double quotes around the `url` for the interpolation correctly, like so:
```
url "https://ghostbrowser.s3.amazonaws.com/GhostBrowser-#{version}.dmg"
```

Comment (straxhaber) [inline Casks/ghost-browser.rb]: @miccal fixed. Let me know if there's anything else.

**Comment by vitorgalvao** [comment]
I don’t see a download for this on the website. Can one only get the download after registering for the beta?

**Comment by miccal** [comment]
your are correct @vitorgalvao - running `brew cask fetch https://raw.githubusercontent.com/straxhaber/homebrew-cask/c145ec42c196b288980db91f6388e16fa1e1029a/Casks/ghost-browser.rb` and running the `.app` requests a login.

**Comment by vitorgalvao** [comment]
It’s not a problem if the app itself requests a login. However, it is a problem if the download URL is hidden, because that is a [walled build](https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#unofficial-vendorless-and-walled-builds) and we do not accept those for security reasons.

**Comment by miccal** [comment]
Ah, of course!

**Comment by miccal** [comment]
Upon further investigation, I believe this is a walled build - the download link is only made available once you have registered for the beta. Although the download link is not hidden _per se_, it is not meant to be available to just anyone.

**Comment by straxhaber** [comment]
Hi @miccal and @vitorgalvao -- yes, I believe you are correct. Thank you for the clarification. I was unaware of the policy but am now. Is this a situation in which we should reach out to the developer of Ghost Browser, or just pass on this?

**Comment by vitorgalvao** [comment]
Lets pass on it *for now*. If the developer wants the URL hidden while in beta, we should respect that.

**Pull request closed without merging**